.solcover.js

```diff
@@ -1,7 +1,4 @@
 module.exports = {
   istanbulReporter: ["html", "lcov"],
-  providerOptions: {
-    mnemonic: process.env.MNEMONIC,
-  },
```

Comment:
This `solcover` option is only used when you use it with ganache, but AFAICT here it's only used with the Hardhat Network, so we removed it.